pkg/runtime/runtime.go

```diff
@@ -903,13 +933,14 @@ func (a *DaprRuntime) initPubSub() error {
 				continue
 			}
 
-			a.scopedSubscriptions = scopes.GetScopedTopics(scopes.SubscriptionScopes, a.runtimeConfig.ID, properties)
-			a.scopedPublishings = scopes.GetScopedTopics(scopes.PublishingScopes, a.runtimeConfig.ID, properties)
-			a.allowedTopics = scopes.GetAllowedTopics(properties)
+			pubsubName := c.ObjectMeta.Name
 
-			a.pubSub = pubSub
+			a.scopedSubscriptions[pubsubName] = scopes.GetScopedTopics(scopes.SubscriptionScopes, a.runtimeConfig.ID, properties)
```

Comment:
this is adding scopes for topics to all pubsub components. @yaron2 Should the scopes be changed to be per pubsub component name as well?

Reply:
Scopes are per pub sub component.

Reply:
Ah yes.

Reply:
This is per pubsub.